update documentation overview extensions (#763)
asreview/asreview: docs/source/extensions/overview_extensions.rst

The following extensions are officially supported and were developed as part
of the core project:

> 
> Model extensions
> ~~~~~~~~~~~~~~~~

Reviewer: ```suggestion

.. _extensions-official-model:

Model extensions
~~~~~~~~~~~~~~~~
```

I think something like this resolves the issue with labels and hyperlinks.

Author: Unfortunately it did not resolve the warning. Since we only have model extensions I suggest to remove the header.

    pip install git@github.com:{USER_NAME}/{REPO_NAME}.github

> To develop your own extenstion go to :ref:`develop-extensions`. 

Reviewer: ```suggestion
See :ref:`develop-extensions` for information about developing your own extension. 
```
